docs/fa/class-dfa.md

```diff
@@ -311,7 +311,7 @@ for word in dfa:
 ## DFA.cardinality(self)
 
 Returns the cardinality of the language represented by the DFA.
-Note that `len(dfa)` raises a `ValueError` for infinite languages, whereas `DFA.cardinality` will return `float('inf')`.
+Note that `len(dfa)` raises a `ValueError` for infinite languages, whereas `DFA.cardinality` will return `None`.
```

Comment:
@eliotwrobson So I think the docs does not match the current implementation (or the other way around 😅. I recently changed the current implementation for `DFA.cardinality` (on develop) raises an `InfiniteLanguageException` in the case of infinite languages (instead of returning `float('inf')`). Not sure what we should do about that.

```python
def cardinality(self):
    """Returns the cardinality of the language represented by the DFA."""
    try:
        i = self.minimum_word_length()
    except exceptions.EmptyLanguageException:
        return 0
    limit = self.maximum_word_length()
    if limit is None:
        raise exceptions.InfiniteLanguageException("The language represented by the DFA is infinite.")
    return sum(self.count_words_of_length(j) for j in range(i, limit+1))
```

Reply:
I think either behavior is fine (since there's a way to check for the language being infinite without the exception getting raised), as long as the docs are consistent with the behavior.

Reply:
@eliotwrobson Okay, then for the sake of backwards-compatibility, I will update the docs to reflect the current (exception-based) behavior. You can revert your docs changes on this PR.

Reply:
Sounds good!